specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2019-09-01/providers.json

```diff
@@ -133,6 +133,13 @@
           "items": {
             "$ref": "#/definitions/LogSpecification"
           }
+        },
+        "metricSpecifications": {
```

Comment:
Why are you adding a new property to a GA'd version? This could cause problems with existing clients, including data loss for input-output models.

/cc @JeffreyRichter

Reply:
This property is reported by the S360 swagger correctness report, which is missing defined in swagger spec in the 2019-09-01 version, but it exists in traffic payload

Reply:
@heaths This property is reported by the S360 swagger correctness report, which is missing defined in swagger spec in the 2019-09-01 version, but it exists in traffic payload